test/check-application

```diff
@@ -791,6 +796,47 @@ class TestFiles(testlib.MachineCase):
         b.click(".breadcrumb-button:nth-of-type(3)")
         m.execute("chattr -i /home/admin/locked")
 
+    def testKeyboardNav(self):
+        b = self.browser
+        m = self.machine
+
+        self.enter_files()
+
+        create_files = ""
+        for i in range(0, 4):
+            create_files += f"touch /home/admin/file{i}; "
+        m.execute(create_files)
+
+        # HACK: Focus the iframe for global keybindings
+        b.eval_js("window.focus()")
+
+        b.click("[data-item='file0']")
+        b.wait_visible("[data-item='file0'].pf-m-selected")
+
+        # b.key_press(chr(39), use_ord=True)  # arrow right
```

Comment:
I suppose that comment is just a leftover now.

Reply:
Jup